scss/_typography.scss

```diff
@@ -65,3 +65,20 @@ ins,
 u {
   text-decoration: underline;
 }
+
+// As per yegor256 suggest https://gist.github.com/unruthless/413930
```

Comment:
@akshayvinchurkar1 you don't need this comment. You just need to mention the issue number in commit messages.